src/pages/stacks-blockchain/running-testnet-node.md

```diff
@@ -91,6 +91,51 @@ INFO [1588108047.585] [src/chainstate/stacks/index/marf.rs:732] First-ever block
 
 Your node will receive new blocks when they are produced, and you can use the [Stacks Node RPC API](/references/stacks-blockchain-api#stacks-node-rpc-api) to send transactions, fetch information for contracts and accounts, and more.
 
+## Running the testnet node on Windows
+
+### Prerequisites
+
+Before you begin, check that you have the below necessary softwares installed on your PC
+
+- [MicrosoftC++BuildTools](https://visualstudio.microsoft.com/visual-cpp-build-tools/).
+- [NodeJs](https://nodejs.org/en/download/).
+- [Git](https://git-scm.com/downloads).
+
+#### Optional Dependencies
+
+- [Python](https://www.python.org/downloads/).
+- [Rust](https://www.rust-lang.org/tools/install).
+
+### Download the Binary and run the follower node
+
+First, download the binary using the below link:
+
+<https://github.com/blockstack/stacks-blockchain/releases/download/v23.0.0.10-krypton/windows-x64.zip>
+
+Extract the binary .Open the command prompt from the folder where binary is extracted and execute the below command:
```

Comment:
"binary .Open" > "binary. Open" (spacing)

-> again, would run a linter. I would assume it would fail with the current sources and we'd want to fix these issues. Failing lint tests would break the deployment of the PR

Reply:
Fixed the linting issues.